welcome/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Parity Ethereum Integration | Parity Technologies</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <script src="./index.js" async></script>
```

Comment:
Can't we have the CSS loaded here ? So there is no FOUC.

Reply:
The resources are loaded from the chrome extension, so it should work really fast and you never should see any UC.
I have inlined the CSS to address that though. (Having a standard css file would require some webpack configuration that I don't think is worth it)